hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/contract/AbstractFSContractTestBase.java

```diff
@@ -272,7 +272,7 @@ protected void handleRelaxedException(String action,
     if (getContract().isSupported(SUPPORTS_STRICT_EXCEPTIONS, false)) {
       throw e;
     }
-    LOG.warn("The expected exception {}  was not the exception class" +
+    LOG.warn("The expected exception {} was not the exception class" +
```

Comment:
IIRC an extra last parameter of type Throwable is allowed for Logger

Reply:
```java
    // org.slf4j.Logger
    public void warn(String msg);
    public void warn(String format, Object arg);
    public void warn(String format, Object... arguments); // <-- I think it will choose this. Maybe in the impl it checks the type to see if it's throwable? 🤷‍♂️ 
    public void warn(String format, Object arg1, Object arg2);
    public void warn(String msg, Throwable t);
    public void warn(Marker marker, String msg);
    public void warn(Marker marker, String format, Object arg);
    public void warn(Marker marker, String format, Object arg1, Object arg2);
    public void warn(Marker marker, String format, Object... arguments);
    public void warn(Marker marker, String msg, Throwable t);
```

Reply:
throwables come at the end, they don't go into the error text